preprocessy/handlenullvalues/_main.py

```diff
@@ -236,6 +214,36 @@ def __categoricalnull(self):
     # main function
     def execute(self, params):
 
+        """Function that handles null values in the supplied dataframe and returns a new dataframe. If no user parameters are supplied, the rows containing null values are dropped by default.
+
+        :param train_df: Input dataframe
+                  Should not be ``None``
+        :type train_df: pandas.core.frames.DataFrame
+
+        :param test_df: Input dataframe
+                  Should not be ``None``
+        :type test_df: pandas.core.frames.DataFrame
+
+        :param cat_cols: List containing the names of categorical columns
+        :type cat_cols: List
+
+        :param replace_cat_nulls: The value which will replace null values in the categorical columns
+        :type replace_cat_nulls: str
+
+        :param drop: Indicates if the null values in the dataframe should be dropped
+        :type drop: bool
+
+        :param fill_missing: Indicates if the null values should be filled with ``mean`` or ``median``
+        :type fill_missing: str
```

Comment:
Add the set of values that this param can have. instead of str, "median" | "mean" and so on